 * Serving Flask app 'app'
 * Debug mode: on
INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
INFO:werkzeug:[33mPress CTRL+C to quit[0m
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 600-993-516
INFO:api_logger:Route: /auth/login Service: unknown_service Method: OPTIONS Status: 200 response_time: 0.00s Timestamp: 2025-08-09 20:03:12 User: anonymous
INFO:werkzeug:127.0.0.1 - - [09/Aug/2025 20:03:12] "OPTIONS /auth/login HTTP/1.1" 200 -
DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): localhost:5001
DEBUG:urllib3.connectionpool:http://localhost:5001 "POST /login HTTP/1.1" 200 273
INFO:api_logger:Route: /auth/login Service: unknown_service Method: POST Status: 200 response_time: 0.52s Timestamp: 2025-08-09 20:03:12 User: anonymous
INFO:werkzeug:127.0.0.1 - - [09/Aug/2025 20:03:12] "POST /auth/login HTTP/1.1" 200 -
INFO:api_logger:Route: /task/tasks/user/manuel Service: unknown_service Method: OPTIONS Status: 200 response_time: 0.00s Timestamp: 2025-08-09 20:03:14 User: anonymous
INFO:werkzeug:127.0.0.1 - - [09/Aug/2025 20:03:14] "OPTIONS /task/tasks/user/manuel HTTP/1.1" 200 -
DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): localhost:5003
DEBUG:urllib3.connectionpool:http://localhost:5003 "GET /tasks/user/manuel HTTP/1.1" 200 997
INFO:api_logger:Route: /task/tasks/user/manuel Service: unknown_service Method: GET Status: 200 response_time: 0.22s Timestamp: 2025-08-09 20:03:15 User: manuel
INFO:werkzeug:127.0.0.1 - - [09/Aug/2025 20:03:15] "GET /task/tasks/user/manuel HTTP/1.1" 200 -
INFO:api_logger:Route: /auth/logs Service: unknown_service Method: OPTIONS Status: 200 response_time: 0.00s Timestamp: 2025-08-09 20:03:16 User: anonymous
INFO:werkzeug:127.0.0.1 - - [09/Aug/2025 20:03:16] "OPTIONS /auth/logs HTTP/1.1" 200 -
DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): localhost:5001
DEBUG:urllib3.connectionpool:http://localhost:5001 "GET /logs HTTP/1.1" 200 217446
INFO:api_logger:Route: /auth/logs Service: unknown_service Method: GET Status: 200 response_time: 0.41s Timestamp: 2025-08-09 20:03:17 User: manuel
INFO:werkzeug:127.0.0.1 - - [09/Aug/2025 20:03:17] "GET /auth/logs HTTP/1.1" 200 -
INFO:api_logger:Route: /task/tasks/user/manuel Service: unknown_service Method: OPTIONS Status: 200 response_time: 0.00s Timestamp: 2025-08-09 20:03:34 User: anonymous
INFO:werkzeug:127.0.0.1 - - [09/Aug/2025 20:03:34] "OPTIONS /task/tasks/user/manuel HTTP/1.1" 200 -
DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): localhost:5003
DEBUG:urllib3.connectionpool:http://localhost:5003 "GET /tasks/user/manuel HTTP/1.1" 200 997
INFO:api_logger:Route: /task/tasks/user/manuel Service: unknown_service Method: GET Status: 200 response_time: 0.08s Timestamp: 2025-08-09 20:03:35 User: manuel
INFO:werkzeug:127.0.0.1 - - [09/Aug/2025 20:03:35] "GET /task/tasks/user/manuel HTTP/1.1" 200 -
INFO:werkzeug: * Detected change in '/mnt/c/Users/manue/Documents/seguridad/microservices/backend/api_gateway/app.py', reloading
INFO:werkzeug: * Restarting with stat
WARNING:werkzeug: * Debugger is active!
INFO:werkzeug: * Debugger PIN: 600-993-516
